### DISCUSSION TITLE
Is there ETA for 0.12?

### DISCUSSION THREAD

**Discussion by denisgurev** [pocketbase/pocketbase#1688] [Q&A] [answered]
Hi, I know that you've mentioned that you don't like to give ETAs but can you at least give some rough estimation when the editor field will be released (https://github.com/pocketbase/pocketbase/issues/370)? I can probably try to build it myself but I don't know go and probably will mess up something 😕.

**Comment by ganigeorgiev** [comment, marked as answer]
It is hard to say, please be patient. I'm still not 100% sure for some of the changes and I'm reluctant to rush the release, but most likely it will be available sometime next week if nothing critical show up.

There are no planned breaking changes to the HTTP APIs in v0.12.0, so you'll be able to upgrade to v0.12.0 by just replacing the binary.
